gui_qt/check_failures_report.py

```
@@ -0,0 +1,583 @@
+            items=display_items,
+            omitted_item_count=omitted_item_count,
+            facts=facts + [f"解析错误：{parse_error}"],
+            detail_lines=detail_lines or [f"解析错误：{parse_error}"],
+        )
+
+    if not report_path:
+        return CheckIssuesReport(
+            status="missing_report",
+            heading="未找到检查报告",
+            message=(
+                "任务清单中没有记录检查报告路径，也无法从翻译包目录推断。"
```

Ruff (0.15.18)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)